packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -93,32 +94,144 @@ export default {
             type: String,
             default: null
         },
-        logo: {
+        logoUrl: {
+            type: String,
+            default: null
+        },
+        imgUrl: {
             type: String,
             default: null
         },
         disabled: {
             type: Boolean,
             default: false
         },
+        tileMode: {
+            type: Boolean,
+            default: true
+        },
         // feature flags
         flags: {
             type: Object,
             default: () => ({})
         }
+    },
+    computed: {
+        displayModeModifier () {
+            return this.tileMode ? 'c-restaurantCard--tile' : 'c-restaurantCard--list-item';
+        }
     }
 };
 </script>
 
 <style lang="scss" module>
+$card-bgColor                             : $color-container-default;
+$card-borderRadius                        : $radius-rounded-c;
+$tile-imgSpacing                          : spacing(x10) * 2;
+$img-border-radius                        : $radius-rounded-c;
+$img-width                                : 156px;
+$logo-borderRadius                        : $radius-rounded-b;
+$logo-borderColor                         : $color-border-default;
+
+@mixin card-frame {
+    z-index: 1;
+    background: $card-bgColor;
+    box-shadow: 0 6px 8px rgba($color-black, 0.02), 0 1px 20px rgba($color-black, 0.08), 0 3px 6px -1px rgba($color-black, 0.08);
```

Comment:
I assume this wouldn't get complained about in our linting rules, but if not, I'd structure this as:

```suggestion
    box-shadow: 0 6px 8px rgba($color-black, 0.02),
                0 1px 20px rgba($color-black, 0.08),
                0 3px 6px -1px rgba($color-black, 0.08);
```